ui/components/app/collectible-details/collectible-details.stories.js

```diff
@@ -9,23 +18,24 @@ export default {
       control: 'object',
     },
   },
+  args: {
+    collectible,
+  },
 };
 
-const collectible = {
-  name: 'Catnip Spicywright',
-  tokenId: '1124157',
-  address: '0x06012c8cf97bead5deae237070f9587f8e7a266d',
-  image: './catnip-spicywright.png',
-  description:
-    "Good day. My name is Catnip Spicywight, which got me teased a lot in high school. If I want to put low fat mayo all over my hamburgers, I shouldn't have to answer to anyone about it, am I right? One time I beat Arlene in an arm wrestle.",
-};
-
-export const DefaultStory = () => {
-  return <CollectibleDetails collectible={collectible} />;
+export const DefaultStory = (args) => {
+  return <CollectibleDetails {...args} />;
 };
 
 DefaultStory.storyName = 'Default';
 
-DefaultStory.args = {
-  collectible,
+export const NoImage = (args) => {
+  return <CollectibleDetails {...args} />;
+};
+
+NoImage.args = {
+  collectible: {
+    ...collectible,
+    image: undefined,
+  },
 };
```

Comment:
Adding another story to display the collectible details story when there is no image